### DISCUSSION TITLE
Is there any limit on the size of the model that can be rendered when the back-end rendering is performed by the tram, or is it related to the hardware? What are the requirements for CPU and graphics card?

### DISCUSSION THREAD

**Discussion by eagle201306** [Kitware/trame#107] [General]
Is there any limit on the size of the model that can be rendered when the back-end rendering is performed by the tram, or is it related to the hardware? What are the requirements for CPU and graphics card?

**Comment by jourdain** [comment]
Hardware specific and if you use ParaView you can run on a cluster across nodes to load even bigger models.
